README.md

```diff
@@ -69,6 +69,8 @@ render() {
   // Supported inline styles: https://github.com/facebook/draft-js/blob/master/docs/Advanced-Topics-Inline-Styles.md
   // Supported block types: https://github.com/facebook/draft-js/blob/master/docs/Advanced-Topics-Custom-Block-Render.md#draft-default-block-render-map
   const toolbarConfig = {
+    hideLinkButtons: true,
+    hideBlockTypeDropdown: false,
```

Comment:
Ah okay cool, I was going to have to test it later today. Glad you're doing it now!